assets/src/components/iconAlertCircle.tsx

```diff
@@ -1,20 +1,41 @@
 import React from "react"
 
-const IconAlertCircle = () => (
+export enum AlertIconStyle {
+  Black = 1,
+  Grey,
+  Highlighted,
+}
+
+const styleClass = (style: AlertIconStyle): string => {
+  switch (style) {
+    case AlertIconStyle.Black:
+      return "--black"
+    case AlertIconStyle.Grey:
+      return "--grey"
+    case AlertIconStyle.Highlighted:
+      return "--highlighted"
+  }
+}
+
+export interface Props {
+  style: AlertIconStyle
+}
+
+const IconAlertCircle = (props: Props) => (
```

Comment:
This is being used by `GhostPropertiesPanel`, does that need to be updated as well?

Reply:
Yeah, it caused a merge conflict, so that fix ended up snuck into 090a2a6 "alert icon controls its own styling"